sos/report/plugins/qaucli.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Qaucli(Plugin, IndependentPlugin):
+
+    short_desc = 'QLogic information'
+
+    plugin_name = 'qaucli'
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('QConvergeConsoleCLI',)
```

Comment:
Will the package name be matched on Debian/Ubuntu? Since we do case sensitive package name check and I see `qconvergeconsolecli` DEB package ("contrary" to `QConvergeConsoleCLI` RPM) in some manuals.

Reply:
I don’t know about that. Worst case if the package name doesn’t match, it won’t execute. I don’t have access to that package nor do I know how on the disk the package extraction looks like. So, I don’t want to add the lowercase package name.

What’s your opinion?

Reply:
> Worst case if the package name doesn’t match, it won’t execute.

Indeed. It isnt a bug, "just" possible lack of new functionality. Until @slashdd or @adam-stokes object (if they know `qconvergeconsolecli` DEB is really in lowercase), I am OK with merging it as is.